src/content/learn/render-and-commit.md

```diff
@@ -1,44 +1,44 @@
 ---
-title: Render and Commit
+title: Рендер і фіксація
 ---
 
 <Intro>
 
-Before your components are displayed on screen, they must be rendered by React. Understanding the steps in this process will help you think about how your code executes and explain its behavior.
+Перш ніж ваші компоненти відобразяться на екрані, вони мають бути відрендереними за допомогою React. Розуміння кроків цього процесу допоможе вам подумати про те, як виконується ваш код, і пояснити його поведінку.
 
 </Intro>
 
 <YouWillLearn>
 
-* What rendering means in React
-* When and why React renders a component
-* The steps involved in displaying a component on screen
-* Why rendering does not always produce a DOM update
+* Що таке рендеринг в React
+* Коли і чому React рендерить компонент
+* Кроки виведення компоненту на екран
+* Чому рендеринг не завжди призводить до оновлення DOM
 
 </YouWillLearn>
 
-Imagine that your components are cooks in the kitchen, assembling tasty dishes from ingredients. In this scenario, React is the waiter who puts in requests from customers and brings them their orders. This process of requesting and serving UI has three steps:
+Уявіть, що ваші компоненти - це кухарі на кухні, які збирають смачні страви з інгредієнтів. У цьому сценарії React - це офіціант, який приймає запити від клієнтів і приносить їм замовлення. Цей процес запиту та обслуговування UI складається з трьох кроків:
 
-1. **Triggering** a render (delivering the guest's order to the kitchen)
-2. **Rendering** the component (preparing the order in the kitchen)
-3. **Committing** to the DOM (placing the order on the table)
+1. **Запуск** рендеру (доставка замовлення гостя на кухню)
```

Comment:
```suggestion
1. **Запуск** рендеру (доставлення замовлення гостя на кухню)
```
https://onlinecorrector.com.ua/uk/%D0%B4%D0%BE%D1%81%D1%82%D0%B0%D0%B2%D0%BB%D1%8F%D0%BD%D0%BD%D1%8F/

Reply:
```suggestion
1. **Запуск** рендеру (передання замовлення гостя на кухню)
```

Reply:
```suggestion
1. **Виклик** (trigger) рендеру (доставка замовлення гостя на кухню)
```